sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -202,6 +205,61 @@ public async Task StartRecognizeReceiptsPopulatesExtractedReceipt(bool useStream
             Assert.That((float?)receipt.Total, Is.EqualTo(1203.39).Within(0.0001));
         }
 
+        /// <summary>
+        /// Verifies that the <see cref="FormRecognizerClient" /> is able to connect to the Form
+        /// Recognizer cognitive service and perform analysis of receipts.
+        /// </summary>
+        [Test]
+        [TestCase(true)]
+        //[TestCase(false)]
```

Comment:
Just curious why this one is commented out?

Reply:
The file is not available in the repo yet (it will be once this PR is merged). I would suggest ignoring the test instead of commenting it out so we don't forget about its existence:
```
[TestCase(false, Ignore="The form has not been uploaded to GitHub yet.")]
```